management_app/templates/faculty/edit.html

```diff
@@ -0,0 +1,131 @@
+  </ul>
+</div>
+<form method="POST" enctype="multipart/form-data" class="col-lg-6 col-sm-12" style="padding-bottom: 60px;">
+  <div class="mb-3">
+    <label class="fw-bold" for="name">Name*</label>
+    <input name="name" type="text" class="form-control" id="name" placeholder="Anteater Smit" value="{{ request.form['name'] or faculty['name'] }}" required>
+  </div>
+  <div class="mb-3">
+    <label class="fw-bold" for="ucinetid">UCI NetID*</label>
+    <input name="ucinetid" type="text" class="form-control" id="ucinetid" placeholder="anteater1" value="{{ request.form['ucinetid'] or faculty['ucinetid'] }}" required>
+  </div>
+  <div class="mb-3">
+    <label class="fw-bold" for="email">Email address*</label>
+    <input name="email" type="email" class="form-control" id="email" placeholder="anteater1@uci.edu" value="{{ request.form['email'] or faculty['email']  }}" required>
+  </div>
+  <div class="mb-3 ">
+    {% set faculty_role = faculty['profile_status']['role'] %}
+    <div class="d-flex">
+      <label class="fw-bold" class="align-self-center" for="role" style="width: 160px;">Faculty Role*&nbsp;</label>
+      <select class="form-select align-self-center" id="role" aria-label="Faculty Role" name="role" required onchange="handleRoleSelection(this)">
+        {% for option, point in role_options.items() %}
+        <option
+          value="{{ option }}" data-point="{{ point }}" id="user_role_option"
+          {% if faculty_role== option %} selected="selected"{% endif %}
+        >
+          {{ option }}
+        </option>
+        {% endfor %}
+      </select>
+    </div>
+    <p>Please update faculty role on the edit page for the latest academic year for faculty member you would like to update.</p>
+    <ul>
+      <li>Required Credit Due: <span id="user_role_credit_due">{{ faculty['credit_due'] }}</span></li>
+    </ul>
+  </div>
+  <div class="mb-3">
+    Faculty Role History in Current System:
+    <ul>
+    {% for f in roles_status %}
+      {% if f['active_status'] %}
+      <li>
+        {{ f['start_year'] }} - {{ f['end_year'] }}: {{ f['role'] }}
+      </li>
+      {% endif %}
+    {% endfor %}
+    </ul>
+  </div>
+  <div class="mb-3">
+    <label class="fw-bold" for="grad_count">Grad Count*</label>
+    <input name="grad_count" type="text" class="form-control" id="grad_count" placeholder="3.33" value="{{ request.form['grad_count'] or faculty['grad_count']  }}" required>
+    <p>If no grad count, please type 0.</p>
+  </div>
+  <div class="mb-3">
+    <label class="fw-bold" for="grad_students">Grad Students</label>
+    <input name="grad_students" type="text" class="form-control" id="grad_students" placeholder="Johnny Wang, John Smith" value="{{ request.form['grad_students'] or faculty['grad_students'] }}">
+    <p>Please use comma to separate students' name.</p>
+  </div>
+  <div class="mb-3">
+    <label class="fw-bold" for="exception_point">Adjust Teaching Points</label>
```

Comment:
It looks a bit misleading to me because this adjust the point balance and has nothing to do with the teaching point calculation. I suggest changing the naming to "Exception Point".
And for the "Reason for Change", I suggest changing the naming to "Reason for Exception" .

These three columns associated with the Exception, so I suggest adding a "Exception" title before them, and then adjust the order: "Exception Category" -> "Reason for Exception" -> "Exception Point" (just like the order in the original data file)

Reply:
I agree with @yuchesu on the first part but for the order I think it should be "Exception Point" -> "Exception Category" -> "Reason for Exception". I think it's more natural to input the exception points and choose the category before typing the reason, but I leave the decision up to you @yrfang.

Reply:
Got it. The labels were updated to "Exception Point" and "Reason for Change". The order "Exception Point" -> "Exception Category" -> "Reason for Exception" looks natural to me, so I leave it for now, but let's confirm with the sponsor. If there is a need to change, let's update the order later.